exercises/grade-school/src/test/java/SchoolTest.java

```diff
@@ -7,96 +7,92 @@
 import java.util.Map;
 import java.util.List;
 import java.util.Collection;
+import java.util.Arrays;
 
 import org.hamcrest.Matcher;
 import org.hamcrest.collection.IsIterableContainingInOrder;
 import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertArrayEquals;
 
-/*
- * Based on problem-specifications/exercises/grade-school/canonical-data.json 1.0.0
- */
 public class SchoolTest {
+  School school;
 
-  @Test
-  public void startsWithNoStudents() {
-    School school = new School();
-    assertThat(school.numberOfStudents(), is(0));
+  @Before
+  public void setUp() {
+    school = new School();
   }
 
-  @Ignore("Remove to run test")
   @Test
   public void addingAStudentAddsThemToTheSortedRoster() {
-    School school = new School();
+    school = new School();
     school.add("Aimee", 2);
-    String[] expected = {"Aimee"};
-    assertArrayEquals(expected, school.roster().toArray(new String[0]));
+    List<String> expected = Arrays.asList("Aimee");
+    assertThat(school.roster(), is(expected));
```

Comment:
A couple of minor things here:
- Since you're creating a list with a single element you can use `Collections.singletonList`
- You should be able to use `assertEquals` instead of `assertThat`. That would be more consistent with our other tests. I know that's not something you've introduced to these tests, just a minor thing that I noticed now 🙂 

Those are both minor things though so I'm happy merging this PR without those changes 🙂 Let me know if you do want to do them as part of this PR, otherwise I'll open a new issue for it 🙂